appserver/tests/payara-samples/samples/pom.xml

```diff
@@ -43,25 +49,7 @@
     </dependencyManagement>
 
     <profiles>
-
-		<profile>
-			<id>unstable</id>
-			<modules>
-		        <module>openid</module>
-		        <module>jaxws-tracing</module>
-		        <module>asadmin</module>
-		        <module>microprofile-endpoints</module>
-         	</modules>
-		</profile>
-		<profile>
-			<id>unstable-remote</id>
-			<modules>
-                <module>ejb-invoker-secure-endpoint</module>
```

Comment:
Ok, these tests are the cause of the slowdown. I will take a look once more if they couldn't be faster (unclosed rest clients or what?)

Reply:
They do many restarts, that's why they are so slow. But that's out of the scope of this issue and I'm not sure if it is even resolvable.